jsonpath2/expressions/operator.py

```diff
@@ -193,6 +193,21 @@ def __evaluate__(x_obj, y_obj):
         return False
 
 
+class InBinaryOperatorExpression(BinaryOperatorExpression):
```

Comment:
Does the `in` consider as BinaryOperator or just Operator?

Reply:
It's the subclass all the other equality checks are using.